presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/AddExchanges.java

```diff
@@ -153,41 +148,29 @@ public PlanNode optimize(PlanNode plan, Session session, Map<Symbol, Type> types
         private final PreferredProperties preferredProperties;
```

Comment:
correlations has nothing to do with this comment.

I would vote for comment removal.